airtime_mvc/application/views/scripts/form/smart-block-criteria.phtml

```diff
@@ -120,7 +120,9 @@
         <?php } ?>
         </dd>
 
-        <?php if ($this->showPoolCount) { ?>
+        <?php
+        // this will never run due to refactoring of playlist controller. it is replaced with code in update.phtml
```

Comment:
Can we remove this whole block then? Since `$this->showPoolCount` is long gone this is just dead code and we can get rid of the whole div without any side effects.